variants/aws-k8s-1.21-nvidia/Cargo.toml

```diff
@@ -0,0 +1,39 @@
+[package]
+# This is the aws-k8s-1.21 variant.  "." is not allowed in crate names, but we
+# don't use this crate name anywhere.
+name = "aws-k8s-1_21-nvidia"
+version = "0.1.0"
+edition = "2018"
+publish = false
+build = "build.rs"
+# Don't rebuild crate just because of changes to README.
+exclude = ["README.md"]
+
+[package.metadata.build-variant]
+included-packages = [
+    "aws-iam-authenticator",
+    "cni",
+    "cni-plugins",
+    "kernel-5.10",
+    "kubelet-1.21",
+    "release",
+    "nvidia-container-toolkit",
+    "kmod-5.10-nvidia-tesla-470"
+]
+kernel-parameters = [
+    "console=tty0",
+    "console=ttyS0",
+]
```

Comment:
Another good one to compare with the original - it's missing the change from 65daf913e.